### DISCUSSION TITLE
Can I only allow Shulker boxes and Ender chests to be built in a container trust?

### DISCUSSION THREAD

**Discussion by nakanotti** [GriefPrevention/GriefPrevention#2147] [Q&A]
I would like to allow the construction and destruction of shulker boxes and ender chests in container trust claims.
The reason is that I want to collect a large amount of items nearby.
Is this possible?

**Comment by QarthO** [comment]
You can create a sub claim just in the small spot you want people to place their shulkers and give everyone build trust by doing /trust public

They’ll be able to place/break any blocks, but only in the small subclaim, and not your farm

**Comment by nakanotti** [comment]
Thank you for your reply.
Unfortunately, it's already in a sub-climb.
It seems difficult to allow specific blocks.
Thank you!
